storybook/pages/TokenListPopupPage.qml

```diff
@@ -89,8 +89,9 @@ SplitView {
                             sourceModel: root.tokensProxyModel
 
                             // Filter by source
-                            filters: ExpressionFilter {
-                                expression: model.jsArraySources.includes(keyFilter.value)
+                            filters: RegExpFilter {
+                                roleName: "sources"
+                                pattern: ".*" + keyFilter.value + ".*"
```

Comment:
this regex is a bit too simple and may generate false matches, to be strict we should look for value bounded by `;` or start of the input in front and `;` or end of the input at end.